build.sbt

```diff
@@ -1428,14 +1428,22 @@ lazy val `runtime-parser` =
     .settings(
       frgaalJavaCompilerSetting,
       instrumentationSettings,
+      commands += WithDebugCommand.withDebug,
+      fork := true,
+      Test / javaOptions ++= Seq(
```

Comment:
That seems new. I guess that will be disappear once you move `persistence` to its own project?